util.go

```diff
@@ -40,13 +39,14 @@ func nFromString(ab string, n int) (string, float64) {
 
 // randomUint32 creates a random 32 bit unsigned integer
 func randomUint32() uint32 {
-	b := make([]byte, 4)
-	_, err := rand.Read(b)
+	var b [4]byte
+	_, err := rand.Read(b[:])
 	if err != nil {
 		panic("PRNG gen error:" + err.Error())
 	}
 
-	return binary.BigEndian.Uint32(b)
+	// Same as binary.BigEndian.Uint32(b[:])
+	return uint32(b[3]) | uint32(b[2])<<8 | uint32(b[1])<<16 | uint32(b[0])<<24
```

Comment:
Hmm. Ok. I accept doing this the Go way with a "little copying". Who am I to argue with Rob Pike?